torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -464,6 +476,7 @@ static c10::optional<MatchedSchema> tryMatchSchema(
   return MatchedSchema{
       std::move(positional_inputs),
       std::move(return_types),
+      {std::move(additions)},
```

Comment:
OOC, why is the curly bracket needed here?

Reply:
Good catch--holdout from when I had a slightly different API

Reply:
Should the curly brackets be removed then?